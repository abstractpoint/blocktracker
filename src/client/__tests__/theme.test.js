import theme from '../theme';

describe('Theme', () => {
  test('Theme should be valid', () => {
    expect(theme).toMatchSnapshot();
  });
  test('Theme should contain breakpoints', () => {
    expect(theme.breakpoints).not.toBe(undefined);
  });
});
