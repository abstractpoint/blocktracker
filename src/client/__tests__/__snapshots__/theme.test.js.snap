// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme Theme should be valid 1`] = `
Object {
  "breakpoints": Array [
    "30rem",
    "50rem",
  ],
  "colors": Object {
    "black": "#000",
    "black-0125": "rgba(0,0,0,.0125)",
    "black-025": "rgba(0,0,0,.025)",
    "black-05": "rgba(0,0,0,.05)",
    "black-10": "rgba(0,0,0,.1)",
    "black-20": "rgba(0,0,0,.2)",
    "black-30": "rgba(0,0,0,.3)",
    "black-40": "rgba(0,0,0,.4)",
    "black-50": "rgba(0,0,0,.5)",
    "black-60": "rgba(0,0,0,.6)",
    "black-70": "rgba(0,0,0,.7)",
    "black-80": "rgba(0,0,0,.8)",
    "black-90": "rgba(0,0,0,.9)",
    "blue": "#357edd",
    "dark-blue": "#00449e",
    "dark-gray": "#333",
    "dark-green": "#137752",
    "dark-pink": "#d5008f",
    "dark-red": "#e7040f",
    "gold": "#ffb700",
    "gray": "#777",
    "green": "#19a974",
    "hot-pink": "#ff41b4",
    "light-blue": "#96ccff",
    "light-gray": "#eee",
    "light-green": "#9eebcf",
    "light-pink": "#ffa3d7",
    "light-purple": "#a463f2",
    "light-red": "#ff725c",
    "light-silver": "#aaa",
    "light-yellow": "#fbf1a9",
    "lightest-blue": "#cdecff",
    "mid-gray": "#555",
    "moon-gray": "#ccc",
    "navy": "#001b44",
    "near-black": "#111",
    "near-white": "#f4f4f4",
    "orange": "#ff6300",
    "pink": "#ff80cc",
    "purple": "#5e2ca5",
    "red": Array [
      "#ff4136",
    ],
    "silver": "#999",
    "washed-blue": "#f6fffe",
    "washed-green": "#e8fdf5",
    "washed-red": "#ffdfdf",
    "washed-yellow": "#fffceb",
    "white": "#fff",
    "white-0125": "rgba(255,255,255,.0125)",
    "white-025": "rgba(255,255,255,.025)",
    "white-05": "rgba(255,255,255,.05)",
    "white-10": "rgba(255,255,255,.1)",
    "white-20": "rgba(255,255,255,.2)",
    "white-30": "rgba(255,255,255,.3)",
    "white-40": "rgba(255,255,255,.4)",
    "white-50": "rgba(255,255,255,.5)",
    "white-60": "rgba(255,255,255,.6)",
    "white-70": "rgba(255,255,255,.7)",
    "white-80": "rgba(255,255,255,.8)",
    "white-90": Array [
      "rgba(255,255,255,.9)",
    ],
    "yellow": "#ffd700",
  },
  "fontSizes": Array [
    ".75rem",
    ".875rem",
    "1rem",
    "1.125rem",
    "1.25rem",
    "1.5rem",
    "1.875rem",
    "2.25rem",
    "3rem",
    "4rem",
  ],
  "spacing": Array [
    0,
    "0.2rem",
    "0.4rem",
    "0.8rem",
    "1.6rem",
    "3.2rem",
    "6.4rem",
    "12.8rem",
  ],
  "typefaces": Object {
    "mono": "Consolas, monaco, monospace",
    "sansSerif": "system-ui, -apple-system, BlinkMacSystemFont, \\"avenir next\\", avenir, \\"helvetica neue\\", helvetica, ubuntu, roboto, noto, \\"segoe ui\\", arial, sans-serif",
    "serif": "athelas, georgia, times, serif",
  },
}
`;
